base_country_iso3166_1_alpha3/models/res_country.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2009 Alejandro Sánchez <alejandro@asr-oss.com>
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+try:
+    import pycountry
+except ImportError:
+    pass
+
+
+class ResCountry(models.Model):
+
+    _inherit = 'res.country'
+
+    @api.multi
+    @api.depends('code')
+    def _compute_code_alpha3(self):
+        for country in self:
+            try:
+                country.code_alpha3 = pycountry.countries.get(
+                    alpha2=country.code).alpha3
+            except KeyError:
+                country.code_alpha3 = ""
```

Comment:
I agree with @ivantodorovich, and I add that empty value must be `False`, not `""`.